### DISCUSSION TITLE
Strongswan Host to Host  scenario for complete LAN encryption

### DISCUSSION THREAD

**Discussion by iattnt22** [strongswan/strongswan#1632] [Q&A]
Greetings
I am looking to encrypt entire traffic on private LAN. I have come across various strongswan scenarios. The example scenario of host to host configuration is hardcoded for two hosts. Suppose I have a private LAN 192.168.1.0/24 and if I want to implement host to host implementation for all traffic between any two hosts, is it possible? 
Thanks

**Comment by tobiasbrunner** [comment]
Have a look at the [ikev2/trap-any](https://strongswan.org/testing/testresults/ikev2/trap-any/) test scenario.
